src/pathfinding_service/service.py

```diff
@@ -84,6 +90,11 @@ def _load_token_networks(self) -> Dict[TokenNetworkAddress, TokenNetwork]:
         channel_views = self.database.get_channel_views()
         for cv in channel_views:
             network_for_address[cv.token_network_address].add_channel_view(cv)
+
+            # Register channel participants for presence tracking
+            self.matrix_listener.follow_address_presence(cv.participant1)
+            self.matrix_listener.follow_address_presence(cv.participant2)
```

Comment:
Is this idempotent? Otherwise, we'll follow most nodes multiple times.

Reply:
Yes:
```py
   def add_address(self, address: Address):
        """ Add ``address`` to the known addresses that are being observed for reachability. """
        # Since _address_to_userids is a defaultdict accessing the key creates the entry
        _ = self._address_to_userids[address]
```